.dockerignore

```diff
@@ -1,37 +1,19 @@
 # list of files to ignore from docker's perspective
 # The more we can exclude the higher likelihood we can reuse cached layers
 
-.assets/
-.circleci/
-.config/
-.github/
-.idea/
-.lintrules/
-developer-docs-site/
-docker/
-!docker/validator/install-tools.sh
-!docker/build-common.sh
-!docker/tools/boto.cfg
-documentation/
-ecosystem/
-!ecosystem/indexer/
-!ecosystem/node-checker/
-scripts/
-target/
-terraform/
-!terraform/helm/aptos-node/
-!terraform/helm/genesis/
+# exclude everything
+**/*
 
-*~
-*.swp
-**/.terraform/
-**/*.md
-**/*.MD
-**/*Dockerfile
-.git/
+# explicitly include stuff we actually need via negation
 
-clippy.toml
-.gitattributes
+!docker/build-rust-all.sh
+!docker/tools/boto.cfg
 
-# Ignore generated credentials from google-github-actions/auth as per https://github.com/google-github-actions/auth#prerequisites
-gha-creds-*.json
+!**/Cargo.toml
+!**/Cargo.lock
+!**/*.rs
+!config/src/config/test_data
+!aptos-move/framework/
+!api/doc/
+!ecosystem/indexer/migrations/**/*.sql
+!terraform/helm
```

Comment:
nit: you can replace this with more specific paths used by Forge. Might result in a few more cache hits
```
!terraform/helm/aptos-node/
!terraform/helm/genesis/
```

Reply:
done